### PR TITLE
v0.5.0 release notes

### PR DESCRIPTION
**Pull request by dcmiddle**

### CODE REVIEW

File: releases/v0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  * s390x does not support cosign signature validation
+- SELinux is not supported on the host and must be set to permissive if in use.
+- Attestation and key brokering support is still under development
+  * The disk-based key broker client (KBC) is used for non-tee testing, but is not suitable for
+  production, except with encrypted VM images.
+  * Currently, there are two key broker services (KBS) that can be used:
+    - simple-kbs:  simple key broker service for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which
+    Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
```
Comment (jialez0) [inline releases/v0.5.0.md:44]: In v0.5.0, generic KBS and supporting KBC can be released. We will integrate them into the guest image/initrd and give the using guide in Quick Start.

cc @sameo @fitzthum @Xynnn007 @jepio

Related issue: https://github.com/kata-containers/kata-containers/issues/6365

File: releases/v0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Process-based isolation is now fully supported with SGX hardware added to enclave-cc CI
+- Improved validation coverage
+
```
Comment (jialez0) [inline releases/v0.5.0.md:14]: Generic KBS (and AS, RVPS) can be released in v0.5.0
```suggestion
+ - Generic key broker system (including [kbs](https://github.com/confidential-containers/kbs), [attestation-service](https://github.com/confidential-containers/attestation-service) and [rvps](https://github.com/confidential-containers/attestation-service/blob/main/docs/rvps.md)) is now supported.
```

Comment (sameo) [reply releases/v0.5.0.md:14]: Thanks @jialez0. Could we expand the acronyms here? Something like:

- Generic Key Broker System (KBS) is now supported. This includes the [KBS](https://github.com/confidential-containers/kbs) itself, which relies on the [Attestation Service (AS)](https://github.com/confidential-containers/attestation-service) for attestation evidence verification. Reference Values are provided to the `AS` by the [Reference Value Provider Service (RVPS)](https://github.com/confidential-containers/attestation-service/blob/main/docs/rvps.md).

**Review by fitzthum — APPROVED** [review]

File: releases/v0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Process-based isolation is now fully supported with SGX hardware added to enclave-cc CI
+- Improved validation coverage
```
[the remark this answers is not in the document]
Comment (dcmiddle) [reply releases/v0.5.0.md:13]: Sure and everyone feel free to use the `Add a suggestion` feature. 

```suggestion
- Improved validation coverage
- SEV configuration can be set with annotations
```

Comment (sameo) [inline releases/v0.5.0.md:13]:
```suggestion
- Improved validation coverage
- Generic Key Broker System (KBS) is now supported. This includes the [KBS](https://github.com/confidential-containers/kbs) itself, which relies on the [Attestation Service (AS)](https://github.com/confidential-containers/attestation-service) for attestation evidence verification. Reference Values are provided to the `AS` by the [Reference Value Provider Service (RVPS)](https://github.com/confidential-containers/attestation-service/blob/main/docs/rvps.md).
```

File: releases/v0.5.0.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  production, except with encrypted VM images.
+  * Currently, there are two key broker services (KBS) that can be used:
+    - simple-kbs:  simple key broker service for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which
+    Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
+- The format of encrypted container images is still subject to change
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
```
Comment (Xynnn007) [inline releases/v0.5.0.md:48]: In v0.5.0, 
- we will have a [unified format of encrypted image for CoCo](https://github.com/confidential-containers/attestation-agent/issues/113)

File: releases/v0.5.0.md (hunk 3 above)
Comment (Xynnn007) [inline releases/v0.5.0.md:13]: Could this be a new feature?
```suggestion
+ Improved validation coverage
+ [KBS Resource URI Scheme](https://github.com/confidential-containers/attestation-agent/blob/main/docs/KBS_URI.md) is raised to identify all confidential resources.
```

File: releases/v0.5.0.md (hunk 4 above)
Comment (dcmiddle) [reply releases/v0.5.0.md:50]: Let's take this item to the next release notes PR. I'm less clear how to handle the text for this one and this PR is starting to get a little messy.

**Review by sameo — APPROVED** [review]

**Pull request merged**